config/redirects.yml

```diff
@@ -394,4 +394,6 @@ redirects:
   "/subjects/physics": "/is-teaching-right-for-me/physics"   
   "/subjects/maths": "/is-teaching-right-for-me/maths"
   "/subjects/engineers-teach-physics": "/is-teaching-right-for-me/engineers-teach-physics"
-  
+
+# Moving qualifications page
```

Comment:
I'm not sure I'd put this under a separate heading in the redirects file as more helpful to bunch similar stuff together - but leave for now as I think we should reorganise the whole redirects file sometime!